core/domain/stats_domain.py

```diff
@@ -1099,9 +1236,11 @@ class StateAnswers:
     """Domain object containing answers submitted to an exploration state."""
 
     def __init__(
-            self, exploration_id, exploration_version, state_name,
-            interaction_id, submitted_answer_list,
-            schema_version=feconf.CURRENT_STATE_ANSWERS_SCHEMA_VERSION):
+            self, exploration_id: str, exploration_version: str,
+            state_name: str, interaction_id: str,
+            submitted_answer_list: List[SubmittedAnswer],
+            schema_version: str=
```

Comment:
I think it's better for this to be an int. Ditto for exploration_version above, since these are numbers.

I'm not sure why the docstring is what it is -- could you PTAL at the callsites and see why they're passing in strings? Versions should typically be numbers so maybe we should fix that unless there's some possible string value here that I'm missing.

Reply:
Done - the callsites pass in ints for both exploration_id and schema_version so I changed the types and docstrings to int.